### PR TITLE
[fix] Fix and refactor

### PR DESCRIPTION
**Pull request by Chiranjibichapagain**
Fixes are from navbar, landing page, modal, and also GoogleUserlogin components. So plz don't kill me.

### CODE REVIEW

File: client/src/components/Navbar/Navbar.scss [hunk 1]
```diff
@@ -36,48 +39,46 @@
...
-  .header {
-    padding: 0.1em 1.5em;
+  .nav {
+    padding: 0.3em 0em;
     height: 70px;
 
-    &__logo {
-      height: 50px;
+    &__image {
+      &--logo {
```
Comment (Rostislav013) [inline client/src/components/Navbar/Navbar.scss:63]: Empty line between lines 62 63 like before

File: client/src/pages/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -66,16 +76,18 @@
 
     &__circle {
       &--left {
```
Comment (Rostislav013) [inline client/src/pages/LandingPage/LandingPage.scss:78]: Add empty line between lines 77 78

**Review by Rostislav013 — APPROVED** [review]

File: client/src/pages/LandingPage/LandingPage.scss (hunk 2 above)
Comment (Chiranjibichapagain) [reply client/src/pages/LandingPage/LandingPage.scss:78]: This automatically happens when i save. cannot fix

File: client/src/components/Navbar/Navbar.scss (hunk 1 above)
Comment (Chiranjibichapagain) [reply client/src/components/Navbar/Navbar.scss:63]: This automatically happens when i save. cannot fix

**Review by MikeVedsted — APPROVED** [review]
Overall, it's fine. 
I don't want you to waste too much time on it but just want to make sure you are aware of how the circles look at different problematic sizes:
overlap
circles

**Pull request merged**